infrastructure/terraform/terraform.tfvars

```diff
@@ -0,0 +1,12 @@
+project_id  = "konomi-ai"
+
+# Pick a region with low spot VM prices. us-west4 is currently the cheapest.
+# https://cloud.google.com/compute/vm-instance-pricing
+region           = "us-west4"
+zone             = "us-west4-a"
+gke_cluster_name = "chronicle-staging"
+number_of_nodes  = 1
```

Comment:
Don't we need at least 2 nodes? one for the front-end/nginx container one for the backend one?

Reply:
Multiple pods can be run on a node, if we use the micro instance we may need more but just wanted to clear up that abstraction.

Reply:
Also looking back, the var `number_of_nodes` actually sets `initial_node_count`, so autopilot should provision it as needed, we don't need to worry too much about the node pools itself in this case.